.github/workflows/pull-request.yaml

```
@@ -98,4 +98,12 @@ jobs:
           cache: yarn
       - run: yarn install --immutable
       - run: yarn build:all
-      - run: yarn test:all
+      - name: Create env file for assisted-ui
+        run: echo "AIUI_APP_API_URL=http://localhost:4173" >> apps/assisted-ui/.env.local
+      - name: Run integration tests
+        uses: cypress-io/github-action@v5
+        with:
+          working-directory: ./libs/integration-tests-ocm
+          browser: chrome
+          config-file: cypress.config.js
+          start: yarn --cwd=../../ preview:assisted-ui
```

See the other comments below...
```suggestion
          start: yarn preview:assisted-ui
```